app/resources/views/pages/home.blade.php

```diff
@@ -40,11 +40,22 @@
 									<i class="fa fa-user"></i>
 								</td>
 								<td class="text-center sign-in-sign-up">
-									Sign in / Sign up
+									Sign in 
 								</td>
 							</tr>
 						</table>
 					</a>
+					<nobr>
```

Comment:
Do you know how to make it work without using "nobr"?

It would be better to avoid it since it is considered "obsolete": https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nobr.